CMakeLists.txt

```diff
@@ -1417,6 +1417,17 @@ if(CONFIG_BUILD_OUTPUT_ADJUST_LMA)
     )
 endif()
 
+if(NOT CONFIG_EXCEPTIONS)
+  set(eh_frame_section ".eh_frame")
+else()
+  set(eh_frame_section "")
```

Comment:
minor nit.

You don't need the `""`.
The difference between `set(eh_frame_section "")` and `set(eh_frame_section)` is that the first creates the var with an empty string as value, whereas the latter clears the var.
There is no behavioral differences between the two in the `foreach()` loop below, but there would be a difference in a test like: `if(DEFINED eh_frame_section)`.

Reply:
thanks for your detailed clarification, I got it.